bioimageio/spec/_internal/io_utils.py

```diff
@@ -272,3 +274,12 @@ def get_sha256(path: Path) -> Sha256:
     sha = h.hexdigest()
     assert len(sha) == 64
     return Sha256(sha)
+
+
+def load_array(source: FileSource) -> NDArray[Any]:
+    local_source = download(source).path
+    return numpy.load(local_source, allow_pickle=False)
```

Comment:
Good that you added `allow_pickle=False`, could you also mention this in the spec docs, so we add a note on how they should save the array?

Maybe a todo for the bioimageio-core is to make sure we load it and export it with `allow_pickle=False` in the `build_model` function.